torch/csrc/jit/import_source.cpp

```diff
@@ -111,20 +95,17 @@ struct ConstantTableValue : public SugaredValue {
   }
 
  private:
-  ArrayRef<at::Tensor> constants_;
+  const std::vector<at::Tensor>* constants_;
 };
 
-// A resolver that doesn't rely on Python, and understands references to model
-// constants.
-struct SourceResolver : public Resolver {
-  explicit SourceResolver(
-      std::shared_ptr<CompilationUnit> cu,
-      size_t version,
-      const std::vector<at::Tensor>& tensor_table)
-      : cu_(std::move(cu)) {
+struct SourceImporterImpl : public Resolver,
```

Comment:
Can we name this SourceResolver? Right now seems like a pimpl thing but it also gets passed around independently as a a resolver, which is confusing.